vorbisgain/vorbisgain_0.37-2.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "vorbisgain",
+    "buildsystem": "autotools",
+    "sources": [
+        {
+            "type": "archive",
+            "url": "http://deb.debian.org/debian/pool/main/v/vorbisgain/vorbisgain_0.37.orig.tar.gz",
+            "sha256": "dd6db051cad972bcac25d47b4a9e40e217bb548a1f16328eddbb4e66613530ec"
+        },
+        {
+            "type": "patch",
+            "path": "0001-temp_files.patch"
```

Comment:
it's not blcking but I think that using `paths` here with an array would be clearer and shorter. (applied in order)

Reply:
:open_mouth: That is so much easier! I will do it right away!